### ISSUE TITLE
Pronto 0.6.0 support

### ISSUE THREAD

**Issue by mmozuras** [siebertm/pronto-coffeelint#6]
Hey, 

I've released `0.6.0` of Pronto this week. There's one breaking change you have to be aware of:

`Runner expects to receive patches/commit via initialize(patches, commit), instead of run(patches, commit).`

Here's an example of how to implement that change in the `Runner`:
https://github.com/mmozuras/pronto-scss/commit/c9fa316d422384f82a26872fbde03a62f2892bf3

You can also now pass the `Runner` class as a parameter to `Message.new`. Here's how:
https://github.com/mmozuras/pronto-rubocop/commit/f19d886831e11ad833cb4dc3c9ba3ab41c392246

I think that it's worth implementing both :smile:

**Comment by ragesoss** [comment]
Is this why it `pronto run` throws an error when `pronto-coffeelint` is installed?

```
$ pronto run
/home/sage/.rvm/gems/ruby-2.1.5/gems/pronto-coffeelint-0.5.0/lib/pronto/coffeelint.rb:6:in `run': wrong number of arguments (0 for 2) (ArgumentError)
```

**Comment by siebertm** [comment]
Yepp. Unfortunately I didn't get to do it till now, but happily accepting
PRs.

Workaround: use pronto ~> 0.5.0 till this is resolved.

Sage Ross notifications@github.com schrieb am Mo., 21. März 2016 um 18:41:

**Comment by siebertm** [comment]
I released 0.6.0 which required pronto ~> 0.6.0
